docs/source/changelog.md

```diff
@@ -6,6 +6,52 @@ command line for details.
 
 ## [Unreleased]
 
+## 2.0
+
+JupyterHub 2.0 is a big release!
+
+The most significant change is the addition of [roles and scopes][rbac]
+to the JupyterHub permissions model,
+allowing more fine-grained access control.
+Read more about it [in the docs][rbac].
+
+In particular, the 'admin' level of permissions should not be needed anymore,
+and you can now grant users and services only the permissions they need, not more.
+We encourage you to review permissions, especially any service or user with `admin: true`
+and consider assigning only the necessary roles and scopes.
+
+[rbac]: ./rbac/index.md
+
+**Due to the change in permissions model,
+upgrading to 2.0 requires shutting down all user servers and re-issuing any tokens**.
```

Comment:
Is it okay to do either of these options?
- shut down servers -> upgrade jh
- upgrade jh -> shut down servers

What do you think happens if a server is running during an upgrade?

Reply:
I think it'll come out okay either way in the end, but shutting down first is better. The options are:

1. shut down (servers unavailable)
2. upgrade
3. start up working servers

or

1. upgrade (server authentication stops working, so *effectively* unavailable, but some caching may leave them accessible for a short time)
2. shut down
3. start up working servers

In either case, the results ought to be the same - servers are unavailable/non-functional until they are restarted. The difference is whether they are shutdown while they aren't working vs running and not working.

Reply:
I think this line would be relevant to convey somehow as an explanation of why servers needs to shut down. Can you try fit that into this text someplace?

> server authentication stops working, so effectively unavailable, but some caching may leave them accessible for a short time